UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -438,13 +438,24 @@ public bool WillInteract(ContextMenuApply interaction, NetworkSide side)
 
 		public void ServerPerformInteraction(ContextMenuApply interaction)
 		{
+			var eyeItem = interaction.Performer.GetComponent<Equipment>().GetClothingItem(NamedSlot.eyes).GameObjectReference;
 			switch (interaction.RequestedOption)
 			{
 				case "Contents":
-					// I think some condition should be met before the user knows what the exact contents of a container are.
-					// Wearing science goggles?
-					ExamineContents();
-					break;
+					{
+						if (eyeItem)
+						{
+							if (!Validations.HasItemTrait(eyeItem, CommonTraits.Instance.ScienceScan))
+							{
+								eyeItem.GetComponent<ReagentScanner>().doScan(interaction.Performer.gameObject, this.gameObject);
+							}
```

Comment:
Is this supposed to be checking false?

Reply:
All the other examples do it like that and if I invert it also inverts the correct logic.

Reply:
Invert it. Its not working right due to this https://github.com/unitystation/unitystation/pull/6810#discussion_r653473823